test/dotnet-projectmodel-server.Tests/Helpers/TestHelper.cs

```diff
@@ -28,8 +29,8 @@ public TestHelper()
             }
 
             _tempPath = CreateTempFolder();
-            var dthTestProjectsFolder = Path.Combine(FindRoot(), "testapp", "DthTestProjects");
-            CopyFiles(dthTestProjectsFolder, _tempPath);
+            _testProjectsDir = Path.Combine(AppContext.BaseDirectory, "TestAssets", "ProjectModelServer");
+            CopyFiles(_testProjectsDir, _tempPath);
```

Comment:
Instead of using the `temp` diretcory you use the TestAssetsManager to manipulate your sample projects - Example - https://github.com/dotnet/cli/blob/rel/1.0.0/test/dotnet-publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs#L47

It is much less code.